pyracing/constants.py

```diff
@@ -245,7 +247,7 @@ def number_to_string(self, number):
             return None
 
 
-class SimSesNum:
+class SimSesId:
```

Comment:
this should maybe be `SimSessionType` or something? It really refers to a type, not an id

Reply:
Haha yeah, I've been going back and forth on this, trying to find a good description. SimSessionType sounds good.